Exiled.API/Features/Scp914.cs

```diff
@@ -64,6 +68,11 @@ public static Scp914Mode ConfigMode
         /// </summary>
         public static bool IsWorking => Scp914Controller._isUpgrading;
 
+        /// <summary>
+        /// Gets a value indicating all GameObject inside the input of Scp914.
+        /// </summary>
+        public static Collider[] Inside914 => Physics.OverlapBox(IntakePosition, Scp914Controller.IntakeChamberSize);
```

Comment:
Maybe something like `InsideIntake` would be more appropriate? This implies the colliders inside 914 in general, not just in the intake.